package.json

```diff
@@ -2,11 +2,11 @@
   "name": "node_js_app",
   "version": "1.0.0",
   "description": "node.js app",
-  "main": "src/index.js",
+  "main": "src/index.ts",
```

Comment:
The main file always should be a JS file. In your case, it's build/index.js it depends on what is your outDir in tsconfig.

Reply:
fixed